book/src/concepts/resource.md

```diff
@@ -36,11 +36,20 @@ Fetching a resource can be done like this:
 ```rust,no_run,noplaypen
 # extern crate amethyst;
 # use amethyst::ecs::{Resources};
-# struct MyResource;
+# #[derive(Debug, PartialEq)]
+# struct MyResource {
+#   pub game_score: i32,
+# }
 # fn main() {
 #   let mut resources = Resources::new();
-// Returns a Option<MyResource>
-let fetched = resources.try_fetch::<MyResource>().expect("No MyResource present in Resources");
+#   let my = MyResource{
+#     game_score: 0,
+#   };
+#   resources.insert(my);
+  // try_fetch returns a Option<Fetch<MyResource>>
+  let fetched = resources.try_fetch::<MyResource>()
+                       .expect("No MyResource present in Resources");
```

Comment:
Would be nice to not panic here, because that's the point of `try_fetch` ;)

Reply:
Good point. I could use 
`If let Some(mut fetched_resource)... else...` 
to replace the expect().
What do you think?

Reply:
Sounds good!

Reply:
Changed :)